web/react/components/sidebar.jsx

```diff
@@ -197,6 +197,19 @@ module.exports = React.createClass({
             if (UserStore.getCurrentId() === msg.user_id) {
                 AsyncClient.getChannels(true);
             }
+        } else if(msg.action === "user_removed") {
+            if(msg.user_id === UserStore.getCurrentId()) {
+                AsyncClient.getChannels(true);
+
+                if(msg.props.channel_id === ChannelStore.getCurrentId() && $('#removed_from_channel').length > 0) {
+                    var channelName = ChannelStore.getCurrent().display_name;
+                    var curUser = UserStore.getProfile(msg.props.remover).username;
+                    $('#removed_from_channel').find('.modal-title').text("Removed from " + channelName);
+                    $('#removed_from_channel').find('.modal-body').children().text(curUser + " removed you from " + channelName);
+
+                    $('#removed_from_channel').modal('show');
```

Comment:
@jwilander Is there a better way to do this? It seems strange to me that we are reaching in and modifying a components internals like this.